src/main/java/com/siawsam/duke/Parser.java

```diff
@@ -0,0 +1,112 @@
+package com.siawsam.duke;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * A parser for user inputs.
+ */
+public class Parser {
+    /**
+     * An array of valid Duke commands to edit tasks.
+     */
+    private static final List<String> COMMANDS = Arrays.asList("done", "delete", "find");
```

Comment:
Perhaps a more intuitive variable name could be used instead of COMMANDS? I was confused as I thought COMMANDS also included add, byte, list etc.